simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderActivity.java

```diff
@@ -526,6 +526,17 @@ else if (id == 1) {
     }
   }
 
+  @Override protected void onPause()
+  {
+    super.onPause();
+
+    final SimplifiedReaderAppServicesType rs = Simplified.getReaderAppServices();
+
+    if (this.book_id != null && this.current_location != null) {
```

Comment:
Is bookmarks how we keep the last-read place in the book?  Would you mind adding a doc string explaining it?

Reply:
Yes I will add that, just not in this PR. I've made a note to myself.